internal/handlers/common/route.go

```diff
@@ -0,0 +1,156 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package common
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/AlekSi/pointer"
+
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/lazyerrors"
+	"github.com/FerretDB/FerretDB/internal/wire"
+)
+
+// Route routes the message.
```

Comment:
I definitely like this name more!

But I'm not sure it belongs in this package. It contains code shared by handlers, but this function is not used by handlers. Placing it in `clientconn` package is probably a better idea?..

Reply:
> I definitely like this name more!
>

Thank you. 

> But I'm not sure it belongs in this package. It contains code shared by handlers, but this function is not used by handlers. Placing it in `clientconn` package is probably a better idea?..

:+1:

Reply:
yes and... I run onto import cycling )